iris_grib/tests/integration/load_convert/test_sample_file_loads.py

```diff
@@ -0,0 +1,223 @@
+# Copyright iris-grib contributors
+#
+# This file is part of iris-grib and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Integration tests for grib2 file loading.
+
+These load various files from iris-test-data, and compare the cube with a
```

Comment:
```suggestion
These tests load various files from iris-test-data, and compare the cube with a
```

A minor change but I found this sentence a bit hard to read. I think it's not clear what you're referring to by only having "These" as the object of the sentence, especially when it is the start of a new paragraph.